### DISCUSSION TITLE
Possible to ignore a Plex user account?

### DISCUSSION THREAD

**Discussion by Kaotac** [mach64/trakt-scrobbler#285] [Q&A] [answered]
Just set up my housemate to be able to watch stuff on Plex and was wondering if there is a config that makes it only the stuff watched on my user account is sent to Trakt?

**Comment by iamkroot** [comment, marked as answer]
https://github.com/iamkroot/trakt-scrobbler/wiki/Players-Setup#user-filter

**Reply by Kaotac** [reply]
I thought I had read it somewhere, I just couldn't for the life of me find it on the wiki. Thank you.
